BridgeAppSDK/SBAConsentReviewStep.swift

```diff
@@ -251,5 +255,29 @@ class SBAConsentReviewStepViewController: ORKPageStepViewController {
         
         return stepResult
     }
+    
+    var consentReviewResult: SBAConsentReviewResult? {
+        return self.result?.results?.findObject({ $0 is SBAConsentReviewResult}) as? SBAConsentReviewResult
+    }
+    
+    // Override the default method for goForward and attempt user registration. Do not allow subclasses
+    // to override this method
+    final override func goForward() {
+        
+        // Check that the user has consented or fail with an error
+        guard let consentResult = self.consentReviewResult where consentResult.isConsented else {
+            let error = NSError(domain: "SBAConsentReviewStepDomain",
+                                code: -1,
+                                userInfo: [NSLocalizedDescriptionKey : Localization.localizedString("SBA_REGISTRATION_NOT_CONSENTED")])
+            self.delegate?.stepViewControllerDidFail(self, withError: error)
+            return
+        }
+        
+        // set the consent to the shared user
+        sharedUser.consentSignature = consentResult.consentSignature
```

Comment:
A valid point.  However, that makes it much harder to remain compatible with AppCore (which I currently need to do b/c mPower uses the `APCUser` object). Until all functionality of Onboarding and user Profile is supported by BridgeAppSDK, this consent flow can only be used to handle default consent.